src/shared/components/locales-dropdown/index.module.css

```diff
@@ -88,5 +88,7 @@
 
     & .show {
         display: block;
+
+        /* height: 100px; */
```

Comment:
remove this?

Reply:
I will update the animation of this dropdown in the future. And I'll probably need this `height` to animate it.